fe/fe-core/src/main/java/com/starrocks/catalog/TableProperty.java

```diff
@@ -65,6 +65,10 @@
      */
     private TStorageFormat storageFormat = TStorageFormat.DEFAULT;
 
+    // This table has been deleted. if hasDelete is false, the BE segment must don't have deleteConditions.
+    // If hasDelete is true, the BE segment maybe have deleteConditions because compaction.
+    private boolean hasDelete = false;
```

Comment:
This property does not need to be persistent. This is because you have already set this value when you replay the delete message. Also, the property of the table will be displayed to the user. It is better to add this property to OlapTable.